_ml-commons-plugin/api.md

```diff
@@ -213,87 +213,87 @@ POST /_plugins/_ml/models/WWQI44MBbzI2oUKAvNUt/_load
 ```json
 {
   "task_id" : "hA8P44MBhyWuIwnfvTKP",
-  "status" : "CREATED"
+  "status" : "DEPLOYING"
 }
 ```
 
-## Unload a model
+## Undeploying a model
 
-To unload a model from memory, use the unload operation.
+To undeploy a model from memory, use the undeploy operation.
 
 ```json
-POST /_plugins/_ml/models/<model_id>/_unload
+POST /_plugins/_ml/models/<model_id>/_undeploy
 ```
 
-### Example: Unload model from all ML nodes
+### Example: Undeploy model from all ML nodes
 
 ```json
-POST /_plugins/_ml/models/MGqJhYMBbbh0ushjm8p_/_unload
+POST /_plugins/_ml/models/MGqJhYMBbbh0ushjm8p_/_undeploy
 ```
 
-### Response: Unload model from all ML nodes
+### Response: Undeploy model from all ML nodes
```

Comment:
```suggestion
### Response: Undeploying model from all ML nodes
```

Reply:
Don't forget the "a": "Deploying a model from all ML nodes" :-)